src/desktop/app/components/navbar/index.tsx

```diff
@@ -201,19 +201,21 @@ export class NavControls extends Component<NavControlsProps> {
         // scroll to anchor element
         let href = (history as any).entries[1].pathname.split('/');
         if(href){
-            href = href[href.length - 1]
+            href = href[href.length - 2] + href[href.length - 1]
```

Comment:
This logic seem bit naive, we have comment links and using split by `/` wouldn't work always I think. For example, https://ecency.com/hive-167922/@claudio83/is-posh-token-underestimated#@acidyo/re-claudio83-qx493l would fail, of course from feed you don't get this link directly, but I want to make sure it works without unexpected issues.